LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -2875,13 +2901,20 @@ MI_Result MI_CALL Pull_SendStatusReport(_In_ LCMProviderContext *lcmContext,
         curl_easy_setopt(curl, CURLOPT_HEADERDATA, &headerChunk);
         curl_easy_setopt(curl, CURLOPT_WRITEDATA, &dataChunk);
 
-        curl_easy_setopt(curl, CURLOPT_SSLCERT, OAAS_CERTPATH);
+        res = curl_easy_setopt(curl, CURLOPT_SSLCERT, OAAS_CERTPATH);
+        if (res != CURLE_OK)
```

Comment:
No logging if any of the other setopt's has an error result?

Reply:
I added in error checks for setopts that we already had error IDs for which is mostly around certificate options. I could add checks for the others but since it's just setting options I figured it wasn't as important and it'll take a lot longer since I'll have to create new string IDs. I was going to wait and consult with Nitin about which ones are really needed.